attributes/default.rb

```diff
@@ -6,4 +6,7 @@
 default['lsyncd']['conf_d'] = '/etc/lsyncd/conf.d'
 default['lsyncd']['log_file'] = '/var/log/lsyncd.log'
 default['lsyncd']['status_file'] = '/var/log/lsyncd-status.log'
-default['lsyncd']['interval'] = 20
+default['lsyncd']['max_delays'] = 1000
+default['lsyncd']['max_processes'] = 1
+default['lsyncd']['interval'] = 10
+default['lsyncd']['insist'] = false
```

Comment:
Hello @andreiocriteo,

could create a namespace with all the settings and provide a template more generic, something like that:
```
default['lsyncd']['settings']['logfile'] = '/tmp/toto'
```
Then use the template to iterate through this hash?